e2e/test/cloudflare204/cloudflare204_test.go

```diff
@@ -0,0 +1,65 @@
+package boot
+
+import (
+	_ "embed"
+	"fmt"
+	"os"
+	"path"
+	"sync"
+	"testing"
+
+	"github.com/internetarchive/Zeno/e2e"
+)
+
+type recordMatcher struct {
+	stoppedOK   bool
+	urlArchived bool
+	errored     bool
+}
+
+func (rm *recordMatcher) Match(record map[string]string) {
+	if record["msg"] == "done, logs are flushing and will be closed" {
+		rm.stoppedOK = true
+	}
+	if record["msg"] == "url archived" && record["url"] == "http://cp.cloudflare.com/" && record["status"] == "204" {
```

Comment:
Could it be worth validating the WARC itself at this point as well? Even just some basic validation (similar to the gowarc tests) could be helpful?

Reply:
I want to do it in another PR, not this PR. It requires changing the code of both zeno and gowarc.